ambari-common/src/main/python/ambari_commons/credential_store_helper.py

```diff
@@ -36,7 +36,7 @@ def removeloglines(lines):
 
 def downloadjar(cs_lib_path, jdk_location):
     # Try to download CredentialUtil.jar from ambari-server resources
-    credential_util_dir = cs_lib_path.split('*')[0].split(":")[-1:][0] # Remove the trailing '*' and get the last directory if an entire path is passed
+    credential_util_dir = cs_lib_path.split('*')[0].split(":")[1:] # Remove the trailing '*' and get the last directory if an entire path is passed
```

Comment:
[1:] returns a list with all elements but the first one. I thought we need the last element. 

This will get the last element.

```python
cs_lib_path.split('*')[0].split(":")[-1]
```